### ISSUE TITLE
Incorrect heading structure on home page

### ISSUE THREAD

**Issue by jvine** [sul-dlss/sul-requests#70]
The text "Administration and mediation tasks for page, scan, and hold/recall requests from SearchWorks." should not be a heading, it's just large text. Please make it a `<p>` or `<div>` or `<span>` and add a class to upsize the text to 1.25em.

![screen shot 2015-04-23 at 4 54 31 pm](https://cloud.githubusercontent.com/assets/6945691/7310008/48b89ab4-e9da-11e4-838a-440411118d7b.png)

**Comment by jkeck** [comment]
@jvine I think we should go over some of the other headings too.  I know I randomly have chosen a heading elsewhere based on what size looked best.  Now that we have a few example forms I think we can do a once-over of all the headings and as long as we stick to that pattern we should be good from here on out.
